java/build/pom.xml

```diff
@@ -103,5 +103,51 @@
             <artifactId>json-unit-assertj</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.jgrapht</groupId>
+            <artifactId>jgrapht-core</artifactId>
+            <version>1.5.2</version>
+        </dependency>
```

Comment:
It doesn't look like this is needed. The Pair class from this library is still used, although you could replace that with your own data structure.